### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by sharathnair9999**
Completed Snackbar component. Please give a revie
snackbar componen
w

### CODE REVIEW

File: pages/docs.html [hunk 1]
```diff
@@ -455,8 +455,134 @@ <h4>Button with Icons</h4>
...
+      ".window-for-baseline .close-icon"
+    );
+    const leadingClose = document.querySelector(".window-for-leading .close-icon");
+    const stackedClose = document.querySelector(".window-for-stacked .close-icon");
+
+    baselineOpen.addEventListener("click", () => {
+      baselineBar.classList.contains("window-for-baseline")
+        ? baselineBar.classList.toggle("hide")
+        : baselineBar.classList.toggle("window-for-baseline");
+    });
```
Comment (dikshadhote) [inline pages/docs.html:559]: Use separate file for js and put js code there

**Review by ajay-mandaviya — APPROVED** [review]

File: pages/docs.html [hunk 2]
```diff
@@ -455,8 +455,134 @@ <h4>Button with Icons</h4>
...
+    stackedOpen.addEventListener("click", () => {
+      stackedBar.classList.contains("window-for-stacked")
+        ? stackedBar.classList.toggle("hide")
+        : stackedBar.classList.toggle("window-for-stacked");
+    });
+
+    stackedClose.addEventListener("click", () => {
+      stackedBar.classList.toggle("hide");
+    });
+
```
Comment (dikshadhote) [inline pages/docs.html:583]: same as above

File: styles/components/card/card.css [hunk 3]
```diff
@@ -32,7 +32,6 @@
 display:flex;
 justify-content:center;
 align-items:flex-start;
-position:relative;
 max-width:18rem;
 flex-direction:column;
 border:0.5px solid #cbd5e1;
```
Comment (dikshadhote) [inline styles/components/card/card.css]: use color variables

File: styles/components/snackbar/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  display: none;
+  transition: display 0.3s ease-in;
+}
+
+.baseline {
+  position: fixed;
+  bottom: 1rem;
+  max-width: 30rem;
+  height: 4rem;
+  padding: 0.1rem 0.3rem;
```
Comment (dikshadhote) [inline styles/components/snackbar/snackbar.css]: use px for values less than 10px

**Review by dikshadhote — CHANGES_REQUESTED** [review]
Overall neat and clean code.
Just make the needful changes

**Pull request merged**